### DISCUSSION TITLE
rdk_doc/Quick_start/install_os/rdk_x5

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#35] [General]
# rdk_doc/Quick_start/install_os/rdk_x5

在使用RDK X5开发板前，需要做下述准备工作。

https://developer.d-robotics.cc/rdk_doc/Quick_start/install_os/rdk_x5

**Comment by CuteEric** [comment]
开发板指示灯一直是绿色，串口信息如下，这是什么问题啊

Hit <SPACE> key to stop autoboot in  2s
Card did not respond to voltage select! : -110
eMMC or SD Card not detected on mmchost 0
switch to partitions #0, OK
mmc1 is current device
eMMC or SD Card detected on mmchost 1
** No partition table - mmc 1 **
Couldn't find partition mmc 1:1
MMC Device 2 not found
no mmc device at slot 2
eMMC or SD Card not detected on mmchost 2
Hobot>

**Reply by xuguosheng110** [reply]
你好，没有识别到sd卡，可以换一下SD卡或者更新miniboot进行验证

**Comment by hiyzoom** [comment]
RDK 3.0能不能把adb功能加回来？RDK 2.1还有adb，3.0就没有了

**Reply by xuguosheng110** [reply]
你好，后续会新发布一个版本，预计在25年一季度

**Comment by mirconn** [comment]
RDK X5 的镜像是不是有问题? balenaEtcher 最后都会报错(image maybe ccorrupt), 我重新下载几次, ,ms5sum 的验证码都一样, 不存在下载的问题, 但是和服务器上的md5 码是不一致.
desktop 和 server 都是一样情况.

**Reply by xuguosheng110** [reply]
你好，可以使用rudus 进行系统的烧录，会更稳定一些

**Reply by MintonLee** [reply]
使用rudus 进行系统的烧录，特别难用！！！

**Comment by Feng2004124** [comment]
推荐使用32G以上的内存卡，我是用16G的内存卡，发现链接显示器，无法显示内容，但是系统应该是正常运行的（黄灯亮起的）

**Comment by xuguosheng110** [comment]
您好，感谢您的来信，我会及时处理！

**Reply by ericlinwk** [reply]
你好,請問可以更新3.1没有? 我正在使用3.0.1，但每次用"sudo apt-get update"都出現“Ign:18 http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy InRelease“，
”Err:19 http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy Release“，”404  Not Found [IP: 47.254.134.193 80]“，“E: The repository 'http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy Release' does not have a Release file.”，還有，RDK Studio不能下載，出現 “Whitelabel Error Page. This application has no explicit mapping for /error, so you are seeing this as a fallback. Sun Jan 05 23:24:12 CST 2025. There was an unexpected error (type=Not Found, status=404)."

**Reply by xuguosheng110** [reply]
目前RDK X5还没有对海外用户开卖，我们的一些包还没有在海外同步，本月内我们会完成源码包的同步，帮助海外用户用起来

**Reply by whf1028** [reply]
我看有3.3.3版本的镜像包了，这个问题是不是解决了

**Comment by ericlinwk** [comment]
你好,請問可以更新3.1没有? 我正在使用3.0.1，但每次用"sudo apt-get update"都出現“Ign:18 http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy InRelease“，
”Err:19 http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy Release“，”404 Not Found [IP: 47.254.134.193 80]“，“E: The repository 'http://archive.d-robotics.cc/ubuntu-rdk-x5 jammy Release' does not have a Release file.”，還有，RDK Studio不能下載，出現 “Whitelabel Error Page. This application has no explicit mapping for /error, so you are seeing this as a fallback. Sun Jan 05 23:24:12 CST 2025. There was an unexpected error (type=Not Found, status=404)."

**Reply by Feng2004124** [reply]
RDK X5 我看了，在24年的11月28号，官网已经更新了3.1，但是你这个问题我不知道怎么解决

**Reply by xuguosheng110** [reply]
你好 我们已经通过cdn映射到了海外，可以直接使用

**Reply by whf1028** [reply]
如何使用的，3.1我尝试解决后，然后系统就挂了，无法开机进入桌面了，现在在重新烧录镜像

**Comment by ipconfiger** [comment]
如果烧录的是Server版的系统，那么如何在烧录的时候就设置好WIFI的SSID和密码，以及默认打开SSH服务以及SSH登陆的账号密码或者账号和密钥

**Comment by xuguosheng110** [comment]
您好，感谢您的来信，我会及时处理！

**Comment by rosfreshman2024** [comment]
烧录镜像时，提示”出了点问题。如果源镜像曾被压缩过，请检查它是否已损坏。Error spawning the child process“，该如何解决呢？

**Reply by maxma615** [reply]
你好，可以检查镜像文件的完整性，更换或更新烧录工具，检查存储设备连接

**Comment by Luyuz** [comment]
如果我想烧录树莓派系统进行调用需要如何配置才可以使用？

**Reply by maxma615** [reply]
推荐烧录RDK官方镜像 https://archive.d-robotics.cc/downloads/os_images/

**Comment by sspa-mxy** [comment]
在进行最后一步启动系统的时候，上电绿灯亮过一会儿就红绿灯都开始亮了

**Reply by WuChao-2024** [reply]
目前建议更新的方式是在可以进入Ubuntu系统中，更新miiniboot后，烧录最新系统，然后在最新系统中再更新一次miniboot，避免系统和miniboot版本差异过大导致无法启动。
https://forum.d-robotics.cc/t/topic/30925

**Comment by xiaoxiaoguaizhanshi** [comment]
烧录好镜像之后，插入在开发板上，上电开发板指示灯没有亮起，什么反应也没有，是什么问题啊。电源适配器还是配套的

**Reply by WuChao-2024** [reply]
目前建议更新的方式是在可以进入Ubuntu系统中，更新miiniboot后，烧录最新系统，然后在最新系统中再更新一次miniboot，避免系统和miniboot版本差异过大导致无法启动。
https://forum.d-robotics.cc/t/topic/30925

**Comment by akipop9** [comment]
我想使用rdk x5部署ethercat主站用于和机器人的关节电机通信,但遇到的重重阻碍: 
1. 使用soem时通信时间高达50ms以上,这个时间延迟根本无法用于机器人控制;
2. 使用igh部署时, 直接无法编译成功;
贵司有没有相关的部署经验? 感觉ethercat是现在机器人控制的主流通信方式之一, 对于这个机器人开发板来说应该是基础功能.

**Reply by xuguosheng110** [reply]
你好，RDK X5 上有 CAN FD 接口，不支持EtherCAT，您有相关的需求，建议了解一下我们的 RDK S100，可以在社区上直接获取到，手册里面也有相关的说明

**Reply by akipop9** [reply]
我搜索了RDK S100的文档https://developer.d-robotics.cc/rdk_doc/rdk_s/RDK,但是并没有找到有关ethercat的描述, 方便给个具体链接的位置吗?

**Reply by WuChao-2024** [reply]
etchercan在S100上大约两周会更新相关说明，请耐心等待。

**Comment by whf1028** [comment]
sd卡通过RDK Studio烧录了ubuntu22.04的镜像，一路执行下去显示成功，现在把SD卡插入到RDK X5开发板上，启动后不是ubuntu22.04系统界面

**Comment by smy0909** [comment]
烧录RDK X5 Mudole的时候是将img文件放在producet文件夹里面的？

**Comment by MintonLee** [comment]
SD卡插入开发板，usb2.0 type c接到PC端，长按Sleep按键（位于耳机接口旁），开发板上电，等待5s，开发板进入烧录模式。

特别难用！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！
特比难用！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！

**Comment by vtyfv** [comment]
RDK X5Module 使用HDMI链接后，打开文件管理器，拖动文件夹，会有重影现象出现，使用的是官方底板，板卡安装也没有问题安装的时候有咔哒声，确定是安装到位了，重启依旧复现这个问题，需如何解决

**Comment by yan-xue-team** [comment]
您好，我想问一下，这x5的开发板是否支持原生ubuntu系统

**Comment by xuguosheng110** [comment]
您好，感谢您的来信，我会及时处理！

**Comment by yan-xue-team** [comment]
x5算力够不够支撑开发利用ros2 nav2进行室内巡检的任务
